apps/arena/mix.exs

```diff
@@ -52,7 +52,8 @@ defmodule Arena.MixProject do
       {:credo, "~> 1.7", only: [:dev, :test], runtime: false},
       {:plug_cowboy, "~> 2.5"},
       {:toxiproxy_ex, "~> 1.1.1"},
-      {:joken, "~> 2.6"}
+      {:joken, "~> 2.6"},
+      {:game_backend, in_umbrella: true}
```

Comment:
Arena release doesn't have :game_backend app deployed. We're basing all our development towards that premise.
I believe we should add a way to develop this feature without this strong dependency. If we do this, the umbrella will lose most of its meaning.

Reply:
I reverted this to a version where we have a `bounties.ex` module in arena with the quest check logic, maybe we're repeating code but it's the best way i found to have this check in arena too, maybe something better would be to have another app that's used as a library to be used in both gamebackend and arena, but i would iterate this, what do you think?

Reply:
I agree. Let's merge but fix later. Create an issue please.

Reply:
Sure thing! issue created here -> https://github.com/lambdaclass/mirra_backend/issues/774